src/styles/tools/_mixins.scss

```diff
@@ -121,7 +121,7 @@
   justify-content: center;
   letter-spacing: 1px;
   outline: 0;
-  padding: 0 16px;
+  padding: 0 16px !important;
```

Comment:
Why important in a mixin? There must be a very strong reason to do so.

Reply:
To add to this, I always make sure to add an explainer comment for `!important` (because there are reasons to use it afterall) to help people understand its purpose when they discover it

Reply:
Yes! Uhm, so adding the `button` class used by the new design system removes the padding from `w-button` in parts of the site that doesn't use the new design.

Reply:
I'll add a comment explaining that now.

Reply:
If this is a temporary fix to resolve the old and the new specificity issues it should all go into _rollout.scss, so that we know to delete it when rollout is complete.